src/caret_analyze/infra/lttng/ros2_tracing/processor.py

```diff
@@ -191,7 +191,7 @@ def __init__(
             self._create_handler(self._handle_rcl_lifecycle_state_machine_init)
 
         self._monotonic_to_system_offset: Optional[int] = monotonic_to_system_time_offset
-        self._caret_init_recorded: defaultdict[int, bool] = defaultdict(lambda: False)
+        self._caret_init_recorded: Dict[int, bool] = defaultdict(lambda: False)
```

Comment:
I guess this modification is not necessary.
```suggestion
        self._caret_init_recorded: defaultdict[int, bool] = defaultdict(lambda: False)
```